top.chidao0000.Watermelon.metainfo.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>top.chidao0000.Watermelon</id>
```

Comment:
`chidao0000.top` doesn't seem to exist. Also it seem that the domain associated with the app is `lceda.cn` so the current value isn't acceptable.

Reply:
also the name `Watermelon` doesn't seem to relate.